src/theme/components/featuredContent.ts

```diff
@@ -0,0 +1,63 @@
+import { wrapperStyles } from "./global";
+
+interface FeaturedContentBaseStyleProps {
+  imagePosition: string;
+  imageWidth: string;
+  isFullWidth: boolean;
+}
+
+const FeaturedContent = {
+  baseStyle: ({
+    imagePosition,
+    imageWidth,
+    isFullWidth,
+  }: FeaturedContentBaseStyleProps) => {
+    let imageAtEnd = imagePosition === "end";
+    let full = isFullWidth === true;
+    let wrapperWidth = "50%";
+    if (imageWidth === "oneQuarter") {
```

Comment:
This works fine for this use case because it's unlikely that new image width values will be added, so you _don't_ need to make an update.
If there's an update and we want 3 new image widths, this may grow unmanageable. Another pattern is to create an object to get the value.

```
const imageWidthsValues = {
  oneQuarter: "25%",
  oneThird: "33%",
  ...
};

let wrapperWidth = imageWidthsValues[imageWidth];
```
(the default should still be set somewhere).